.github/workflows/build-cli.yml

```diff
@@ -60,12 +62,20 @@ jobs:
       - name: Build CLI
         env:
           CROSS_NO_WARNINGS: 0
+          RUST_LOG: debug
```

Comment:
were these left for testing? or do we want them here?

Reply:
helps to find issues